shlr/grub/fs/ext2.c

```diff
@@ -385,7 +385,11 @@ grub_ext2_read_block (grub_fshelp_node_t node, grub_disk_addr_t fileblock)
 
   if (grub_le_to_cpu32(inode->flags) & EXT4_EXTENTS_FLAG)
     {
-      char buf[EXT2_BLOCK_SIZE(data)];
+#ifndef _MSC_VER
+	  char buf[EXT2_BLOCK_SIZE (data)];
```

Comment:
grub_malloc is a wrapper for malloc. and the  char buf[EXT2_BLOCK_SIZE (data)]; is the original code form this file, i set the conditional compilation why else we need revise all code to made the free().